src/core/contract_storage.mli

```diff
@@ -0,0 +1,11 @@
+open Tezos
+open Smart_contracts
+type t [@@deriving yojson]
+
+val empty : t
+val equal : t -> t -> bool
+val originate_contract :
+  t -> address:Contract_hash.t -> contract:Contract.t -> t
```

Comment:
Deku contract should not be using Tezos.Contract_hash, it's not a Tezos.Contract_hash